packit/constants.py

```diff
@@ -40,6 +40,10 @@
 
 PROD_DISTGIT_HOSTNAME = "src.fedoraproject.org"
 PROD_DISTGIT_URL = f"https://{PROD_DISTGIT_HOSTNAME}/"
+
+STG_DISTGIT_HOSTNAME = "src.stg.fedoraproject.org"
+STG_DISTGIT_URL = f"https://{STG_DISTGIT_HOSTNAME}/"
```

Comment:
The same comment as above: wouldn't it be easier to make this part of the configuration?